common/expression/tests/it/testdata/run-ast.txt

```diff
@@ -2,119 +2,466 @@ raw expr       : and(true::Boolean, false::Boolean)
++-----------+-------------------------------------------------------------------------------------------------------------------+
+
 
 raw expr       : get_tuple(1)(ColumnRef(0)::Nullable<(Boolean, String)>{not_null})
 type           : Nullable<String>
 property       : {not_null}
 checked expr   : get_tuple<Nullable<(Boolean, String)>>(ColumnRef(0){not_null})
-column[0]      : Nullable { column: Tuple { fields: [Boolean([0b___00000]), String { data: [97, 98, 99, 100, 101], offsets: [0, 1, 2, 3, 4, 5] }], len: 5 }, validity: [0b___00011] }
-eval result    : Nullable { column: String { data: [97, 98, 99, 100, 101], offsets: [0, 1, 2, 3, 4, 5] }, validity: [0b___00011] }
+input chunk:
++------------+
+| Column 0   |
++------------+
+| (false, a) |
+| (false, b) |
+| NULL       |
+| NULL       |
+| NULL       |
++------------+
+input chunk (internal):
++-----------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Column ID | Column Data                                                                                                                                                          |
++-----------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| 0         | Nullable { column: Tuple { fields: [Boolean([0b___00000]), String { data: [97, 98, 99, 100, 101], offsets: [0, 1, 2, 3, 4, 5] }], len: 5 }, validity: [0b___00011] } |
++-----------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+evaluation result:
++----------+
+| Column 0 |
++----------+
+| a        |
+| b        |
+| NULL     |
+| NULL     |
+| NULL     |
++----------+
+evaluation result (internal):
++-----------+-------------------------------------------------------------------------------------------------------------------+
+| Column ID | Column Data                                                                                                       |
++-----------+-------------------------------------------------------------------------------------------------------------------+
+| 0         | Nullable { column: String { data: [97, 98, 99, 100, 101], offsets: [0, 1, 2, 3, 4, 5] }, validity: [0b___00011] } |
++-----------+-------------------------------------------------------------------------------------------------------------------+
+
 
 raw expr       : create_array()
 type           : Array<Nothing>
 property       : {not_null}
 checked expr   : create_array<>()
-eval result    : EmptyArray
+evaluation result:
+[]
+
 
 raw expr       : create_array(NULL, true::Boolean)
 type           : Array<Nullable<Boolean>>
```

Comment:
`Array(X)` may be better than `Array<X>`, the former follows the SQL Parser.
So does `Nullable(x)`